coderd/workspaces.go

```diff
@@ -670,39 +668,14 @@ func (api *API) watchWorkspace(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	c, err := websocket.Accept(rw, r, &websocket.AcceptOptions{
-		// Fix for Safari 15.1:
-		// There is a bug in latest Safari in which compressed web socket traffic
-		// isn't handled correctly. Turning off compression is a workaround:
-		// https://github.com/nhooyr/websocket/issues/218
-		CompressionMode: websocket.CompressionDisabled,
-	})
+	err := httpapi.SetupSSE(rw, r)
```

Comment:
Do we not use this endpoint anywhere yet? I would've expected that you'd need to change frontend code.